mix.exs

```diff
@@ -23,6 +23,7 @@ defmodule PhoenixEcto.Mixfile do
 
   def application do
     [
+      mod: {Phoenix.Ecto, []},
```

Comment:
Hi @chrismccord! Awesome work!

One question, is it OK to run `Phoenix.Ecto.SQL.SandboxSupervisor` in all the environments, including production? I wonder if it would be better to start the supervisor explicitly when running the tests. What do you think?

Reply:
It is totally fine. A process that does nothing won't be of any harm.

Reply:
Yeah, it won’t :)

My concerns were related to people observing their system in development and/or production and seeing processes especifically designed for testing. If I would see that, I would wonder what the value of `MIX_ENV` env var is :sweat_smile: